wallet/db.c

```diff
@@ -912,7 +912,10 @@ static struct migration dbmigrations[] = {
 	 ", PRIMARY KEY(in_channel_scid, in_htlc_id))"), NULL},
     {SQL("INSERT INTO forwards SELECT"
```

Comment:
Putting here just a FIXME with your reason? I think I will take on this task after the release!